tests/unit/js/examples.js

```diff
@@ -0,0 +1,70 @@
+const fieldset = `<fieldset id="identifiers">
```

Comment:
Rename this to `editionIdentifiersSample` or something; the more specific the better (within reason :P). You want a future person reading this to not have to open this file to figure out what is being imported.

`import { fieldset } from examples` gives me no clue as to what's happening.
`import { editionIdentifiersSample } from 'html-test-data.js'` tells me much more.
Similarly with `$(document.body).html(examples.fieldset);` and `$(document.body).html(htmlTestData.editionIdentifiersSample);`

> Remember that you (and others after you) will be reading the code many hundreds of times, but only writing it a few times. Take the time to spell out `connectionPool` instead of `cp`. 
--Pragmatic Programmer

Reply:
Done